crates/biome_js_analyze/src/lint/nursery/no_qwik_react_props.rs

```diff
@@ -0,0 +1,111 @@
+use crate::JsRuleAction;
+use biome_analyze::context::RuleContext;
+use biome_analyze::{
+    Ast, FixKind, Rule, RuleDiagnostic, RuleDomain, RuleSource, declare_lint_rule,
+};
+use biome_console::markup;
+use biome_diagnostics::Severity;
+use biome_js_factory::make::{jsx_ident, jsx_name};
+use biome_js_syntax::{AnyJsxAttributeName, JsxAttribute};
+use biome_rowan::{AstNode, BatchMutationExt, TextRange};
+use biome_rule_options::no_qwik_react_props::NoQwikReactPropsOptions;
+
+declare_lint_rule! {
```

Comment:
We already posses such a rule: https://biomejs.dev/linter/rules/no-react-specific-props/

Just add the `Qwick` domain to the existing rule. Maybe let's update the docs of the current rule, make it a bit more generic.